src/autopas/containers/linkedCells/LinkedCells.h

```diff
@@ -287,12 +287,6 @@ class LinkedCells : public ParticleContainer<ParticleCell, SoAArraysType> {
    */
   std::vector<ParticleCell> &getCells() { return this->_cells; }
 
-  /**
-   * @copydoc getCells()
-   * @note const version
-   */
-  const std::vector<ParticleCell> &getCells() const { return this->_cells; }
-
```

Comment:
is this not needed? I'd leave it, as it makes sort of sense...

Reply:
This is provided by the parent class. Thus this function only shadows an in fact identical function. Now that I see it, we should move the non-const version to the interface too.